README.md

```diff
@@ -62,15 +66,17 @@ end
           password: System.get_env("MYSQL_PASSWORD"),
           database: "my_db_shard_1",
           hostname: "10.0.0.1",
-          pool_size: 15
+          pool_size: 15,
+          priv: "priv/shards"
```

Comment:
Can we make this key a little more explicit? Maybe `migrations_path`?

Reply:
The `priv` key comes from Ecto.  I would prefer to maintain Ecto's naming to avoid confusion.  The `priv` directory also holds the seeds and schema files in addition to the migrations folder.